dwarf/expr.cc

```diff
@@ -41,9 +41,12 @@ expr::evaluate(expr_context *ctx, const std::initializer_list<taddr> &arguments)
         // Create the initial stack.  arguments are in reverse order
         // (that is, element 0 is TOS), so reverse it.
         stack.reserve(arguments.size());
-        for (const taddr *elt = arguments.end() - 1;
-             elt >= arguments.begin(); elt--)
-                stack.push_back(*elt);
+        if (arguments.size() > 0)
```

Comment:
Please follow the style of the surrounding code. The curly brace should be on the same line as the `if` and indentation is 8 spaces.